### PR TITLE
Add kube-fetch documentation

### PR DESCRIPTION
**Pull request by ddgenome**

### CODE REVIEW

File: docs/pack/kubernetes/sync.md [hunk 1]
```diff
@@ -218,6 +218,108 @@ slight load on the Kubernetes master.  Note that applying the all the
...
+Kubernetes cluster, you can use the `--options-file` command-line
+option to provide the path to a file containing the selection options
+for which resources to fetch.
+
+```
+atomist kube-fetch --options-file=OPTIONS_FILE
+```
+
+The `OPTIONS_FILE` should contain a JSON object with the following
+structure.
```
Comment (jessitron) [inline docs/pack/kubernetes/sync.md]: is there an options file they could start from, that we could link?

Comment (ddgenome) [reply docs/pack/kubernetes/sync.md]: No, but I could include an example. The default options include selectors with the `filter` property, which is a function and therefore cannot be specified in a JSON object read from a file.

Comment (ddgenome) [reply docs/pack/kubernetes/sync.md]: That reminds me, you can't include regular expressions in a JSON object either.

Comment (ddgenome) [reply docs/pack/kubernetes/sync.md]: I've added an example and removed the regular expression stuff.

**Pull request merged**